CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
```

Comment:
This commit (3538b2ef30096f81428ea8497eca40f149635044) should not be embedded in this PR as it has a dedicated one (#1416)

Reply:
Yes I was waiting for it to be merged to rebase :pray: